test/e2e/service_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// TestClusterIPHostAccess tests traffic from host to Cluster IP Service.
+func TestClusterIPHostAccess(t *testing.T) {
+	// TODO: Support for dual-stack and IPv6-only clusters
+	skipIfIPv6Cluster(t)
+
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	svcName := "nginx"
+	node := nodeName(0)
+	svc, cleanup := data.createClusterIPServiceAllInOne(t, svcName, node)
+	defer cleanup()
+	time.Sleep(3 * time.Second) // Wait a while for the rules to be installed by kube-proxy/antrea-proxy.
```

Comment:
It may be better to have a retry mechanism for the `curl`. I don't remember if this is something `curl` itself provides or something for which we can use `wait.PollImmediate`. What do you think?

Reply:
Indeed. I used `curl --connect-timeout 1 --retry <svc-ip>:80` to retry curl.
I chose `sleep` before because I saw there're some other places in e2e code where sleep is used after a Service is created. Maybe this is also an improvement issue for e2e.